* relax test for exact match, compare difference with predefined tolerance instead
author: Miguel Dias Costa (National University of Singapore)
--- test/tools/dptools/test_grids.py.orig	2017-10-27 14:16:30.679276000 +0800
+++ test/tools/dptools/test_grids.py	2017-10-27 14:17:28.896727649 +0800
@@ -39,7 +39,8 @@
         )
         gridcoords = grid.cartesian_to_gridcoord([1.9, -0.5, 1.5])
         true_gridcoords = np.array([-1, 2, 9])
-        self.assertTrue(np.all(gridcoords == true_gridcoords))
+        diff = np.max(np.abs(gridcoords - true_gridcoords))
+        self.assertLess(diff, FLOAT_TOLERANCE)
 
 
     def test_get_corners_gridcoord(self):
